src/Model/GServer.php

```diff
@@ -1714,22 +1893,34 @@ private static function detectFriendica(string $url, array $serverdata): array
 	 *
 	 * @param object $curlResult result of curl execution
 	 * @param array  $serverdata array with server data
-	 * @param string $url        Server URL
+	 *
 	 * @return array server data
 	 */
-	private static function analyseRootBody($curlResult, array $serverdata, string $url): array
+	 private static function analyseRootBody($curlResult, array $serverdata): array
 	{
 		if (empty($curlResult->getBody())) {
 			return $serverdata;
 		}
 
-		// Using only body information we cannot safely detect a lot of systems.
-		// So we define a list of platforms that we can detect safely.
-		$valid_platforms = ['friendica', 'friendika', 'diaspora', 'mastodon', 'hubzilla', 'misskey', 'peertube', 'wordpress', 'write.as'];
+		if (file_exists(__DIR__ . '/../../static/generator.config.php')) {
```

Comment:
Please use a more specific name for this file, like `platforms.vars.php` or something like this. In particular `*.config.php` end up returning the config array they define, while this file doesn't return a single structure.

Reply:
Okay, renamed.